src/FSharpVSPowerTools.Logic/GoToDefinitionFilter.fs

```diff
@@ -46,13 +46,24 @@ type GoToDefinitionFilter(textDocument: ITextDocument,
     let urlChanged = if fireNavigationEvent then Some (Event<UrlChangeEventArgs>()) else None
     let mutable currentUrl = None
 
+    let getCurrentFilePathProjectAndDoc () =
+        maybe {
+            let filepath = textDocument.FilePath
+            let dte = serviceProvider.GetService<EnvDTE.DTE, SDTE>()
+            let! doc = dte.GetCurrentDocument(filepath)
+            let! project = maybe {
```

Comment:
This `maybe` computation expression block is entirely redundant. We can move it to the top level.